SpeechService/Humanize.cs

```diff
@@ -128,9 +145,29 @@ private static string FormatWith2SignificantDigits(int number, bool isNegative,
             }
         }
 
+        private static CultureInfo _formattingCultureInfo;
+        private static CultureInfo formattingCultureInfo
+        {
+            get
+            {
+                if (_formattingCultureInfo == null)
+                {
+                    _formattingCultureInfo = (CultureInfo)CultureInfo.CurrentUICulture.Clone();
+                    if (Properties.FormatOverrides.overrideThousandsSeparator.Equals("true"))
```

Comment:
Same as above. Consider adding the secondary optional argument `StringComparison.InvariantCultureIgnoreCase` to `.Equals(string value, StringComparison comparisonType)` to make this user entered parameter a little more forgiving?

Reply:
Answer as above.

Reply:
Response as above. :-)